packages/devtools-core/src/App/UserAppContainer.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import styled from "styled-components";
+import { dispatch } from "../model";
+import { eventListener } from "../utils/communication";
+
+const StyledUserAppContainer = styled.div`
+  flex-grow: 1;
+  text-align: center;
+`;
+
+const StyledIFrame = styled.iframe`
+  min-height: 100vh;
+  height: 100%;
+  min-width: 50%;
+
+  border: none;
+`;
+
+interface Props {
+  url?: string;
+  children?: React.ReactNode;
+}
+
+const UserAppContainer = (props: Props) => {
+  const iFrameRef = React.useRef<HTMLIFrameElement>(null);
+  const [updateValue, forceUpdate] = React.useState(true);
+
+  // Forces a re-render (forceUpdate replacement)
```

Comment:
Not sure you need "forceUpdate repalcement"

Reply:
I do need something there, otherwise the iframe doesn't load reliably. But I can try debugging a bit more to see exactly why.

Reply:
Sorry, I was talking about the comment :D. `// Forces a re-render` is probably good. I was just not sure what `(forceUpdate replacement)` meant

Reply:
Haha that makes sense!